### ISSUE TITLE
Great Job!

### ISSUE THREAD

**Issue by TieKnotMath** [praeclarum/CuneiformTranslators#5]
I want to make this easier to use. My intention is to create an Image-to-text translation on cuneiform tables. People could go to a Turkish market and Transliterate the tables. Also we could add some historic integrations, for example if its talking about a known king or something like that.

**Comment by praeclarum** [comment]
Yes, absolutely, I've been working on this very problem but have been delayed.

I am building a tool to gather the data I need. If you're interested in contributing to that data collection effort, I could really use the help.

**Comment by TieKnotMath** [comment]
When do we start?
